src/tree/FunctionTreeItem.ts

```diff
@@ -73,32 +71,42 @@ export class FunctionTreeItem implements ILogStreamTreeItem {
     }
 
     public async initializeTriggerUrl(): Promise<void> {
-        const kuduClient: KuduClient = await getKuduClient(this.client);
-        const functionSecrets: FunctionSecrets = await kuduClient.functionModel.getSecrets(this._name);
-        if (functionSecrets.triggerUrl === undefined) {
-            throw new ArgumentError(functionSecrets);
+        const triggerUrl: URL = new URL(`${this.client.defaultHostUrl}/api/${this._name}`);
+        const key: string | undefined = await this.getKey();
+        if (key) {
+            // tslint:disable-next-line:no-backbone-get-set-outside-model
```

Comment:
Given that you're not using backbone/, you should probably just disable this rule